docs/excel/custom-functions-overview.md

```diff
@@ -297,7 +297,9 @@ As you can see, ranges are handled in JavaScript as arrays of row arrays (like a
 
 ## Changelog
 
-- **Nov 7, 2017**: Shipped the custom functions preview and samples
+- **Nov 7, 2017**: Shipped* the custom functions preview and samples
 - **Nov 20, 2017**: Fixed compatibility bug for those using builds 8801 and later
-- **Nov 28, 2017**: Shipped support for cancellation on asynchronous functions (requires change for streaming functions)
-- **May 7, 2018**: Shipped support for Mac, Excel Online, and synchronous functions running in-process
+- **Nov 28, 2017**: Shipped support* for cancellation on asynchronous functions (requires change for streaming functions)
+- **May 7, 2018**: Shipped support* for Mac, Excel Online, and synchronous functions running in-process
+
+* - to the Office Insiders Channel
```

Comment:
This `*` displays as a bullet instead of an asterisk.

Reply:
You need to add an escape character (/) if you want it to look like an asterisk.